tests/test_main.py

```diff
@@ -12,22 +12,57 @@
     write_groups_information,
     write_students_information,
     read_groups_information,
-    read_students_information
+    read_students_information,
 )
 
 
+@pytest.fixture
+def specialty_init():
+    return Specialty("Math and Physic", 1)
+
+
+@pytest.fixture
+def student_init():
+    return Student("Ivan", "Ivanov", date.today(), 87.5, True, "+380999999999", "Kyiv")
```

Comment:
I believe this fixture should use `basic_student` function

Reply:
Oh, yes, should